docs/socorro_test_checklist.rst

```diff
@@ -0,0 +1,123 @@
+======================
+Socorro Test Checklist
+======================
+
+This is a high-level system-wide checklist for making sure Socorro is working
+correctly in a specific environment. It's helpful after we've made a significant
+change.
+
+.. contents::
+
+
+Collector
+=========
+
+Is the collector web process handling incoming crashes?
+
+    * Log into collector node and watch the collector logs for errors.
+
+Is the collector crashmover process saving crashes to S3? ES? Postgres?
+RabbitMQ?
+
+    * Log into collector node and watch the crashmover logs for errors.
+
+      FIXME: List log paths.
+
+    * Check datadog ``crashmover.save_raw_crash`` for the appropriate
+      environment.
+
+      :prod: it's on the Socorro Prod dashboard
+      :stage: ?
+  
+    * Submit a crash. Verify raw crash made it to S3. See these
+      docs:
+      http://socorro.readthedocs.io/en/latest/configuring-socorro.html?highlight=curl#test-collection-and-processing
+   
+
+Processor
+=========
+
+Is the processor process running?
+
+    * Log into processor node and watch the processor logs for errors.
+
+      FIXME: List log paths.
```

Comment:
``/var/log/messages`` is the log file. You have to be ``root`` to read it. I usually ``grep ERROR /var/log/messages | less`` to check for errors.